nautobot/dcim/models/devices.py

```diff
@@ -408,12 +430,65 @@ class Platform(OrganizationalModel):
     )
     description = models.CharField(max_length=200, blank=True)
 
+    @cached_property
+    def _network_drivers(self):
+        NETWORK_DRIVERS = deepcopy(get_settings_or_config("NETWORK_DRIVERS"))
+        for key, mapping in (
+            ("ansible", ANSIBLE_LIB_MAPPER_REVERSE),
+            # TODO ("hier_config", HIER_CONFIG_LIB_MAPPER_REVERSE),
+            ("netmiko", NETMIKO_LIB_MAPPER_REVERSE),
+            ("ntc_templates", NTCTEMPLATES_LIB_MAPPER_REVERSE),
+            ("pyats", PYATS_LIB_MAPPER_REVERSE),
+            ("pyntc", PYNTC_LIB_MAPPER_REVERSE),
+            ("scrapli", SCRAPLI_LIB_MAPPER_REVERSE),
```

Comment:
Should this be rewritten to allow the `NETWORK_DRIVERS` setting to provide more tool names (dict keys) than these? I'm noticing nornir is absent here.

Reply:
Nornir does not have a native driver, you can leverage napalm, netmiko, pyntc, scrapli with it though.

Reply:
Wouldn't ansible be the same since it also leverages netmiko under the hood?

Reply:
Ansible "can" use netmiko (or napalm, or pyntc), but it has it's own driver.